packages/slate-react/src/components/editor.js

```diff
@@ -252,22 +236,6 @@ class Editor extends React.Component {
     })
   }
 
-  /**
-   * On change.
-   *
-   * @param {Change} change
-   */
-
-  onChange = change => {
```

Comment:
We can't remove this `editor.onChange(change)` function because it's part of the public API that people are depending on. At least for now.

Reply:
`onChnage` is restored in 37502ee9